scripts/scr_powers_new/scr_powers_new.gml

```diff
@@ -1,56 +1,57 @@
-function scr_powers_new(argument0, argument1) {
+function scr_powers_new(){
 
-	// Argument0: Company
-	// Argument1: Marine ID
+	// company: Company
+	// marine_number: Marine ID
 
 	// This script checks if the marine is capable of using psychic powers, and if so, assigns them based on experience
 
 	var random_learn;
 	random_learn=false;
 
-	var let,letmax,powers_should_have,powers_have;
-	let="";letmax=0;
+	var powers_should_have,powers_have;
```

Comment:
```suggestion
	var letmax, powers_should_have, powers_have;
```
`letmax` is assigned to and used still, so removal of the var declaration will cause that variable to get injected into the callers scope inadvertently.